ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/AbstractProblemFiller.java

```diff
@@ -21,7 +25,7 @@ public AbstractProblemFiller(LinearRaoProblem linearRaoProblem, LinearRaoData li
 
     public abstract void fill();
 
-    public void update(LinearRaoProblem linearRaoProblem, LinearRaoData linearRaoData) {
+    public void update(LinearRaoProblem linearRaoProblem, LinearRaoData linearRaoData, List<RemedialActionResult> remedialActionResultList) {
```

Comment:
I would have removed the todo, declared the method as abstract, and implement it as empty in the positiveMinMarginFiller

Reply:
ok